build.sbt

```diff
@@ -21,6 +21,7 @@ lazy val commonDependencies = Seq(
   Dependencies.Libraries.jodaTime,
   Dependencies.Libraries.slf4j,
   Dependencies.Libraries.log4jOverSlf4j,
+  Dependencies.Libraries.log4j,
```

Comment:
@dilyand I understood that log4j is only bundled in the nsq collector.  So no need to have it here in commonDependencies, but better to just add it to the app that uses it.

Reply:
Yes, good catch, I moved it.